### PR TITLE
Updated index.rst file

### PR DESCRIPTION
**Pull request by orifox**
Removed Collapsing Cubes and made link to JDox.

### CODE REVIEW

**Comment by javerbukh** [comment]
Is there a reason you deleted the `quickstart.rst` file and the collapsing cubes file?

**Comment by orifox** [comment]
Jesse, first one was a typo (quickstart.rst) is supposed to still be there.  Thanks.  Second one was intentional.  I though we were moving collapsing cubes into the plugins section.  See outline here: https://innerspace.stsci.edu/display/DATB/Outline+of+jdaviz+documentation

**Comment by javerbukh** [comment]
I understand the motivation behind linking to https://jwst-docs.stsci.edu/jwst-post-pipeline-data-analysis, however, that page links to the old (QT) versions of specviz, mosviz, and cubebviz. Once that page is updated to link to the jdaviz repo, I think we can merge this PR. As it stands, I think this will add confusion to people reading through the documentation thoroughly.

**Comment by orifox** [comment]
Ok, the JDox page is updated!  Please take a look and merge :)

File: README.rst [hunk 1]
```diff
@@ -16,6 +16,13 @@ JDAViz
...
+.. note::
+
+   ``Jdaviz`` is one tool that is part of STScI's larger `Data Analysis Tools Ecosystem <https://jwst-docs.stsci.edu/jwst-post-pipeline-data-analysis>`_.
+
+.. note::
+
+   ``Jdaviz`` has complete `documentation <https://jdaviz.readthedocs.io/en/latest/>`_.
 
 ``jdaviz`` is a package of astronomical data analysis visualization
 tools based on the Jupyter platform.  These GUI-based tools link data
```
Comment (javerbukh) [inline README.rst:28]:
```suggestion
``jdaviz`` is a package of astronomical data analysis visualization tools based on the Jupyter platform. It is one tool that is a part of STScI's larger `Data Analysis Tools Ecosystem <https://jwst-docs.stsci.edu/jwst-post-pipeline-data-analysis>`_. These GUI-based tools link data
```

**Review by javerbukh — CHANGES_REQUESTED** [review]
Left a comment. I think the change to the RTD index page is good but the change to the README file do not appear as expected. My suggestions remove the `.. note::` line since that is a Sphinx tool reference to the documentation, which the user can access by clicking the badge already in the README.

**Comment by orifox** [comment]
Ok, thanks. I was hoping that would get caught in the review process.  Perfect.  Thanks for the change.

**Review by javerbukh — APPROVED** [review]
Looks good to me!

**Pull request merged**